main.go

```diff
@@ -1,7 +1,13 @@
 package main
 
-func main()  {
-	i:=0
-	i+=1
+func Add() int {
+	i := 0
+	i += 1
+	return i
+}
+
+func main() {
+	i := 0
+	i += 1
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>should replace `i += 1` with `i++` (golint)